wurst/_handles/Destructable.wurst

```diff
@@ -1,8 +1,11 @@
 package Destructable
 import NoWurst
-import Vectors
 import Wurstunit
 import Annotations
+import initlater Unit
```

Comment:
don't add initlater import. Also this stuff does not look like it belongs in stdlib. Put it into some own package and then you also don't need initlater.

Reply:
I could add a package called ElevatorWalls, but where could be good place it?

Reply:
In your own project, not the stdlib.

Reply:
ok

Reply:
Very well, I removed it.